core/chains/evm/logpoller/log_poller_internal_test.go

```diff
@@ -287,7 +287,9 @@ func TestLogPoller_Replay(t *testing.T) {
 	db := pgtest.NewSqlxDB(t)
 	orm := NewORM(chainID, db, lggr)
 
+	headMutex := sync.RWMutex{}
 	head := evmtypes.Head{Number: 4}
```

Comment:
Would it be a litter cleaner to load and store an atomic pointer?
```suggestion
	var head atomic.Pointer[evmtype.Head]
	head.Store(&evmtypes.Head{Number: 4})
```

Reply:
Definitely! Should have thought of that.